protocol-tokio/src/protocol/codec/handshake.rs

```diff
@@ -43,6 +43,12 @@ impl de::Deserialize for ProtocolMagic {
     }
 }
 
+impl From<&u32> for ProtocolMagic {
```

Comment:
Why does it need to be a reference? `u32` is copyable.

Reply:
Fixed